convert to origami 2.0 component (#71)
Financial-Times/o-stepped-progress: README.md

> 
> ### Silent mode
> 

Reviewer: ```suggestion
```

> As with all Origami components, o-stepped-progress has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than incorporating its mixins into your own Sass) call `@include oSteppedProgress();` in your Sass after you import the o-stepped-progress Sass:

Reviewer: ```suggestion
To output all styles call `@include oSteppedProgress();` in your Sass:
```
